examples/throughput/main.go

```diff
@@ -95,6 +100,57 @@ func write(sess net.Conn, numBytes int) error {
 	return nil
 }
 
+func readUDP(conn *udp.EncryptUDPConn, numBytes int) error {
+	defer conn.Close()
+	buffer := make([]byte, 1024)
+	timeStart := time.Now()
+	for {
+		n, _, err := conn.ReadFromUDP(buffer)
```

Comment:
`EncryptUDPConn` is missing at least 5 methods:
Read()
Write()
SetDeadline()
SetReadDeadline()
SetWriteDeadline()

Here we should use `SetReadDeadline` before each read to make it stop when no data is being received